sections/collage.liquid

```diff
@@ -5,8 +5,8 @@
 
 {%- style -%}
   .section-{{ section.id }}-padding {
-    padding-top: calc({{ section.settings.padding_top }}px * 0.75);
-    padding-bottom: calc({{ section.settings.padding_bottom }}px  * 0.75);
+    padding-top: {{ section.settings.padding_top | times: 0.75 | round: 0 }}px;
```

Comment:
The only remaining question I have here is about the rounding. I don't think we need it 🤔  Since basically we're saying that the mobile value will be`3/4` of the desktop one and the steps for the padding settings is set to be incremented by 4, it's always going to be either a whole number or a `.5` as a result but nothing else.

Reply:
My logic was two-fold for adding the rounding.

1. It removes the `.0` from the rendered result, which feels cleaner and if things do need to round, it will cut off unnecessary (and unpredictable) decimals.
2. If we change the way the padding scales (either changing the multiplier or otherwise) or the way the range works (steps change from 4, etc...), we've got the rounding in place.  This one could be argued against with YAGNI, but since it also cleans up the rendered result in point 1, I think it's okay to keep.  
 
It doesn't add any complexity to the code, really, anyway.